package.json

```
@@ -24,5 +24,10 @@
   "dependencies": {
     "@stencil/core": "^2.0.1"
   },
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
+    "jest": "^27.0.6",
+    "puppeteer": "~10.0.0"
```

The latest version of Stencil doesn't support v10 of Puppeteer just yet. Although it's landed in the codebase, I can't guarantee a specific date we'll release Stencil v2.7.0 just yet. Would you mind downgrading this in the mean time?
```suggestion
    "puppeteer": "^5.5.0",
    "@types/puppeteer": "^5.4.3"
```
(since we're downgrading puppeteer, we also need to add `@types/puppeteer`)